src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1833,7 +1833,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Type defines operator == or operator != but does not override Object.GetHashCode()</value>
   </data>
   <data name="ERR_OutAttrOnRefParam" xml:space="preserve">
-    <value>Cannot specify only Out attribute on a ref parameter. Use both In and Out attributes, or neither.</value>
+    <value>Cannot specify Out attribute on a ref parameter without also specifying In attribute.</value>
```

Comment:
`Cannot specify the Out attribute on a ref parameter without also specifying the In attribute.` (add `the` before `Out` and `In`).